concepts/literals/introduction.md

```diff
@@ -36,7 +36,7 @@ This is a good default case and use-cases with the narrower `float` type are les
 ```cpp
 auto light_year_in_m{9.46073e+15f}; // well in the range of float
 auto earth_to_edge_comoving_distance_in_nm{4.32355e+32}; // needs double type for magnitude
-auto eulers_number{2.718281828459045d}; // needs double type for precision
+auto eulers_number{2.718281828459045}; // needs double type for precision
```

Comment:
Did you remove it, because floating-points are resolved to doubles by default?

Reply:
Yeah, I should have probably opened a separate PR for this. The `d` is suffix is not part of the [C++ standard](https://en.cppreference.com/w/cpp/language/floating_literal) it seems:

> suffix, if present, is one of f, l, F, L, f16, f32, f64, f128, bf16, F16, F32, F64, F128, BF16(since C++23).

ChatGPT claims it was a common thing, but I have never seen it before and apparently it is only supported by GCC. Clang and MSVC [report an error](https://godbolt.org/z/fGK55nsM8).

Reply:
Ah, I checked it with GCC. Good catch! I should run it through other compilers. I will add some words on the standard that will be deducted when nothing is added.

Reply:
@vaeng  Can we merge this and create a ticket for adding some sentences that you mentioned, so it doesn't fall through the cracks?

Reply:
Yes, let's do this.